task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -33,11 +35,18 @@ spec:
       image: registry.access.redhat.com/ubi8/ubi
       script: |
         #!/usr/bin/env bash
-        echo "Querying Pyxis for $(params.IMAGE_REPOSITORY)..."
-        http_code=$(curl -s -k -o $(workspaces.sanity-ws.path)/repository_data.json -w '%{http_code}' "https://catalog.redhat.com/api/containers/v1/repositories/registry/$(params.IMAGE_REGISTRY)/repository/$(params.IMAGE_REPOSITORY)")
+        IFS=$'\n' read -d'' -r -a IMAGE_ARRAY < <(echo -n $BASE_IMAGES_DIGESTS | sed 's/\\n/\'$'\n''/g' && printf '\0')
+        for BASE_IMAGE in ${IMAGE_ARRAY[@]};
+        do
```

Comment:
If you use a `do` like this to loop the repos, the last result will be saved only.

Reply:
if results.PYXIS_HTTP_CODE.path is a file then it should be appended

Reply:
Ok, thanks.